gno.land/cmd/gnoland/testdata/assertorigincall.txtar

```diff
@@ -56,61 +48,31 @@ stdout 'OK!'
 ! gnokey maketx call -pkgpath gno.land/r/foo -func C -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1
 stderr 'invalid non-origin call'
 
-## remove due to update to maketx call can only call realm (case 7,8,9)
-## 7. MsgCall -> p/demo/bar.A -> myrlm.A: PANIC
-## ! gnokey maketx call -pkgpath gno.land/p/demo/bar -func A -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1
-## stderr 'invalid non-origin call'
-
-## 8. MsgCall -> p/demo/bar.B -> myrlm.B: PASS
-## gnokey maketx call -pkgpath gno.land/p/demo/bar -func B -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1
-## stdout 'OK!'
-
-## 9. MsgCall -> p/demo/bar.C -> myrlm.C: PANIC
-## ! gnokey maketx call -pkgpath gno.land/p/demo/bar -func C -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1
-## stderr 'invalid non-origin call'
-
-## 10. MsgRun -> run.main -> myrlm.A: PANIC
+## 7. MsgRun -> run.main -> myrlm.A: PANIC
 ! gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/myrlmA.gno
 stderr 'invalid non-origin call'
 
-## 11. MsgRun -> run.main -> myrlm.B: PASS
+## 8. MsgRun -> run.main -> myrlm.B: PASS
 gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/myrlmB.gno
 stdout 'OK!'
 
-## 12. MsgRun -> run.main -> myrlm.C: PANIC
+## 9. MsgRun -> run.main -> myrlm.C: PANIC
 ! gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/myrlmC.gno
 stderr 'invalid non-origin call'
 
-## 13. MsgRun -> run.main -> foo.A: PANIC
+## 10. MsgRun -> run.main -> foo.A: PANIC
 ! gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/fooA.gno
 stderr 'invalid non-origin call'
 
-## 14. MsgRun -> run.main -> foo.B: PASS
+## 11. MsgRun -> run.main -> foo.B: PASS
 gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/fooB.gno
 stdout 'OK!'
 
-## 15. MsgRun -> run.main -> foo.C: PANIC
+## 12. MsgRun -> run.main -> foo.C: PANIC
 ! gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/fooC.gno
 stderr 'invalid non-origin call'
 
-## 16. MsgRun -> run.main -> bar.A: PANIC
-! gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/barA.gno
-stderr 'invalid non-origin call'
-
-## 17. MsgRun -> run.main -> bar.B: PASS
-gnokey maketx run -gas-fee 100000ugnot -gas-wanted 4000000 -broadcast -chainid tendermint_test test1 $WORK/run/barB.gno
-stdout 'OK!'
```

Comment:
Why did these get removed?

Msgrun should be able to import packages.

Reply:
I remove it because it tested the flow: msgRun -> run.main -> package -> realm
But indeed, i should have keep the flow with just removing the link to the realm
fix here: 21f15fe